truffle/src/com.oracle.truffle.api.test/src/com/oracle/truffle/api/test/polyglot/MultiThreadedLanguage.java

```diff
@@ -52,11 +52,11 @@
 @TruffleLanguage.Registration(id = MultiThreadedLanguage.ID, name = MultiThreadedLanguage.ID)
 public class MultiThreadedLanguage extends TruffleLanguage<LanguageContext> {
 
-    static final String ID = "MultiThreadedLanguage";
+    public static final String ID = "MultiThreadedLanguage";
 
-    static final ThreadLocal<Function<Env, Object>> runinside = new ThreadLocal<>();
+    public static final ThreadLocal<Function<Env, Object>> runinside = new ThreadLocal<>();
```

Comment:
runinside does not have to be public

Reply:
Fixed in 75e60df. No longer needed because we removed `org.graalvm.polyglot.Context.Builder` API and its associated test.